### PR TITLE
Review

### PR DESCRIPTION
**Pull request by vdukhovni**
Work in progress.  Please review carefully. I hope the changes are clear and correct. There's more to do...

### CODE REVIEW

File: DANE-for-SMTP-how-to.md [hunk 1]
```diff
@@ -43,16 +43,50 @@ DANE is designed to work with any TLS service, not just email, but DANE for HTTP
...
-* By default mail servers do not validate the authenticity of another mail server's certificate; any random certificate is accepted (see [RFC 2487](https://tools.ietf.org/html/rfc2487). This was probably done because there is no user who can act on errors in case they occur. Unfortunately this default behavior introduces the risk of a man in the middle attack; an attacker can offer a false certificate enabling the attacker to decrypt encrypted traffic.
+* By default mail servers do not validate the authenticity of another mail server's certificate; any random certificate is accepted (see [RFC 3207](https://tools.ietf.org/html/rfc3207).
```
Comment (dennisbaaten) [inline DANE-for-SMTP-how-to.md]:
```suggestion
* By default mail servers do not validate the authenticity of another mail server's certificate; any random certificate is accepted (see [RFC 3207](https://tools.ietf.org/html/rfc3207)).
```

File: DANE-for-SMTP-how-to.md [hunk 2]
```diff
@@ -43,16 +43,50 @@ DANE is designed to work with any TLS service, not just email, but DANE for HTTP
...
+    - It was unclear which CAs to trust to validate the certificate for a given destination.
+    - As a result of MX indirection, it was unclear which names to verify in the certificate.
```
Comment (dennisbaaten) [inline DANE-for-SMTP-how-to.md]: I'm unfamiliar with the concept of "MX indirection". Can you elaborate on this?

File: DANE-for-SMTP-how-to.md [hunk 3]
```diff
@@ -43,16 +43,50 @@ DANE is designed to work with any TLS service, not just email, but DANE for HTTP
...
-* By default mail servers do not validate the authenticity of another mail server's certificate; any random certificate is accepted (see [RFC 2487](https://tools.ietf.org/html/rfc2487). This was probably done because there is no user who can act on errors in case they occur. Unfortunately this default behavior introduces the risk of a man in the middle attack; an attacker can offer a false certificate enabling the attacker to decrypt encrypted traffic.
+* By default mail servers do not validate the authenticity of another mail server's certificate; any random certificate is accepted (see [RFC 3207](https://tools.ietf.org/html/rfc3207).
+    - It was unclear which CAs to trust to validate the certificate for a given destination.
```
Comment (dennisbaaten) [inline DANE-for-SMTP-how-to.md]:
```suggestion
    - It was unclear which CAs to trust when validating the certificate for a given destination (and no user to interact with for manual validation)
```

File: DANE-for-SMTP-how-to.md (hunk 1 above)
Comment (vdukhovni) [reply DANE-for-SMTP-how-to.md]: Sure.

File: DANE-for-SMTP-how-to.md (hunk 2 above)
Comment (vdukhovni) [reply DANE-for-SMTP-how-to.md]: Expanding the wording.

File: DANE-for-SMTP-how-to.md (hunk 3 above)
Comment (vdukhovni) [reply DANE-for-SMTP-how-to.md]: Sure.

**Pull request merged**